sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -719,13 +723,17 @@ public Mono<Void> openConnectionsAndInitCaches() {
             CosmosContainerProactiveInitConfig proactiveContainerInitConfig =
                 new CosmosContainerProactiveInitConfigBuilder(Collections.singletonList(cosmosContainerIdentity))
                     .setProactiveConnectionRegionsCount(1)
+                    .withMinConnectionsPerEndpointForContainer(cosmosContainerIdentity, Configs.getMinConnectionPoolSizePerEndpoint())
                     .build();
 
-            return withContext(context -> openConnectionsAndInitCachesInternal(proactiveContainerInitConfig)
-                                            .flatMap(openResult -> {
-                                                logger.info("OpenConnectionsAndInitCaches: {}", openResult);
-                                                return Mono.empty();
-                                            }));
+            return withContext(context -> openConnectionsAndInitCachesInternal(
+                    proactiveContainerInitConfig,
+                    AsyncDocumentClient.OpenConnectionAggressivenessHint.AGGRESSIVE
```

Comment:
Do we need to pass the aggressiveness here? the only places we need to control the concurrency is in the openConnectionProcessor right?

Reply:
Can be removed.